docs/content/users/quickstart.md

```diff
@@ -323,19 +323,21 @@ ddev launch
 
 ## Shopware
 
-You can set up a Shopware 6 environment many ways, we recommend the following technique:
+You can set up a Shopware 6 environment many ways, we recommend the following technique. DDEV creates a .env.local file for you by default; if you already have one DDEV leaves it alone.
```

Comment:
I’d add a “though” or a “while” so there’s a relationship between those clauses, and ideally clarify _why_ this is the technique we recommend. Also wrap filenames in backticks:

```suggestion
Though you can set up a Shopware 6 environment many ways, we recommend the following technique. DDEV creates a `.env.local` file for you by default; if you already have one DDEV leaves it alone.
```